modules/gsp-cluster/data/values.yaml

```diff
@@ -117,6 +114,18 @@ harbor:
     tlsCA: ${notary_ca_pem}
     tlsCert: ${notary_cert_pem}
     tlsKey: ${notary_root_key}
+  database:
+    type: external
+    external:
+      host: "${harbor_db_host}"
+      port: "${harbor_db_port}"
+      username: "${harbor_db_username}"
+      password: "${harbor_db_password}"
+      coreDatabase: "registry"
+      clairDatabase: "clair"
+      notaryServerDatabase: "notary_server"
+      notarySignerDatabase: "notary_signer"
+      sslmode: "prefer"
```

Comment:
`prefer` is the default for libpq, by the looks of things. It isn't clear where it will go looking for the CA certs; so I thought "get something that works first" and we can iterate later.

Reply:
I believe `require` is the sslmode that means "I still want to use TLS even if I can't trust the CA"

https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-SSLMODE-STATEMENTS